src/index.html

```diff
@@ -21,7 +21,322 @@
     />
   </head>
 
-  <body>
-    <h1>Catalog</h1>
+  <body class="body">
+    <header class="header">
+      <div class="header__container">
+        <a
+          href="#"
+          class="header__logo"
+        >
+          <img
+            src="/src/images/logo.png"
+            alt="Moyo-Logo"
+          />
+        </a>
+        <nav class="nav">
+          <ul class="nav__list">
+            <li class="nav__item">
+              <a
+                href="#"
+                class="nav__link is-active"
+              >
+                Apple
+              </a>
+            </li>
+            <li class="nav__item">
+              <a
+                href="#"
+                class="nav__link"
+              >
+                Samsung
+              </a>
```

Comment:
Add empty lines between multiline sibling blocks of HTML
![image](https://github.com/mate-academy/layout_catalog/assets/63243136/48af5715-dc79-4bdb-b3c5-05cfeb385a3f)

The same for other sibling blocks